### PR TITLE
Fix package names and dependencies

### PR DESCRIPTION
**Pull request by vidurananayakkara**
Dependencies fixed according to the new secure vault changes

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -197,18 +203,24 @@
     </build>
     <properties>
         <!-- Version ranges -->
-        <slf4j.logging.package.import.version.range>[1.7.1, 2.0.0)</slf4j.logging.package.import.version.range>
-        <osgi.framework.package.import.version.range>[1.8.0, 2.0.0)</osgi.framework.package.import.version.range>
-        <org.snakeyaml.package.import.version.range>[1.17.0,2.0.0)</org.snakeyaml.package.import.version.range>
+        <slf4j.logging.package.import.version.range>[1.7.1, 2.0.0)
```
Comment (thusithathilina) [inline pom.xml]: IMHO better to format this to a single line

File: samples/org.wso2.carbon.config.samples.osgi/pom.xml [hunk 2]
```diff
@@ -52,8 +52,8 @@
         </export.package>
         <import.package>
             org.osgi.framework.*;version="${osgi.framework.package.import.version.range}",
-            org.wso2.carbon.secvault.securevault.*,
-            org.wso2.carbon.config.*,
+            org.wso2.carbon.secvault.component.*;version="${carbon.securevault.version.range}",
+            org.wso2.carbon.config.*;version="${carbon.config.package.import.version.range}",
```
Comment (thusithathilina) [inline samples/org.wso2.carbon.config.samples.osgi/pom.xml]: Why do we need import config?

File: tests/org.wso2.carbon.config.maven.plugin.test/src/main/java/org/wso2/carbon/config/maven/plugin/test/ConfigDocumentMojoTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+/*
+*  Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thusithathilina) [inline tests/org.wso2.carbon.config.maven.plugin.test/src/main/java/org/wso2/carbon/config/maven/plugin/test/ConfigDocumentMojoTest.java]: Do we need to change the year?

File: tests/org.wso2.carbon.config.test.coverage/pom.xml [hunk 4]
```diff
@@ -77,28 +78,34 @@
...
                                     <target xmlns:jacoco="antlib:org.jacoco.ant">
-                                        <taskdef uri="antlib:org.jacoco.ant" resource="org/jacoco/ant/antlib.xml">
+                                        <taskdef uri="antlib:org.jacoco.ant"
+                                                 resource="org/jacoco/ant/antlib.xml">
                                             <classpath path="${project.build.directory}"/>
                                         </taskdef>
                                         <jacoco:report>
                                             <executiondata>
-                                                <fileset dir="../../components/org.wso2.carbon.config/target">
+                                                <fileset
```
Comment (thusithathilina) [inline tests/org.wso2.carbon.config.test.coverage/pom.xml]: Better to format as single lines

File: samples/org.wso2.carbon.config.samples.osgi/pom.xml (hunk 2 above)
Comment (vidurananayakkara) [inline samples/org.wso2.carbon.config.samples.osgi/pom.xml]: This OSGi sample will be removed. There will only be a standalone configuration sample.

File: pom.xml (hunk 1 above)
Comment (vidurananayakkara) [inline pom.xml]: Changed to a single line

File: tests/org.wso2.carbon.config.maven.plugin.test/src/main/java/org/wso2/carbon/config/maven/plugin/test/ConfigDocumentMojoTest.java (hunk 3 above)
Comment (vidurananayakkara) [inline tests/org.wso2.carbon.config.maven.plugin.test/src/main/java/org/wso2/carbon/config/maven/plugin/test/ConfigDocumentMojoTest.java]: Year changed to 2017

File: tests/org.wso2.carbon.config.test.coverage/pom.xml (hunk 4 above)
Comment (vidurananayakkara) [inline tests/org.wso2.carbon.config.test.coverage/pom.xml]: Changed to a single line

**Pull request merged**